DOL.WHD.Section14c.Web/env.js

```diff
@@ -1,13 +1,9 @@
 (function(window) {
   window.__env = window.__env || {};
-
-  // base api url
-  window.__env.api_url = 'https://localhost:44399';
-
-  // Google reCaptcha Site Key
-  window.__env.reCaptchaSiteKey = '6LeqeggUAAAAALC5zT4OHbDJk9gHNT0GGZbJMOnG';
-
-  window.__env.requireHttps = true;
-
-  window.__env.tokenCookieDurationMinutes = 20160;
+  window.__env = {
+    api_url: 'https://localhost:44399',
+    reCaptchaSiteKey: '6LeqeggUAAAAALC5zT4OHbDJk9gHNT0GGZbJMOnG',
+    requireHttps: true,
+    tokenCookieDurationMinutes: 20160
+  };
```

Comment:
reformatted to remove redundant `window.__env` prefixing

Reply:
If `window.__env` already exists, this change will overwrite it entirely, whereas the previous code modifies an existing one.  Maybe this would be a good place for `Object.assign`?  E.g.,

```js
Object.assign(window.__env, {
  api_url: '',
  ...
});
```